DEVELOPMENT.md

```diff
@@ -63,7 +63,20 @@ which you can open with your web browser.
 ### Examples
 
 You can check that the [examples](https://github.com/cleanlab/examples) still
-work with changes you make to cleanlab by manually testing them.
+work with changes you make to cleanlab by manually running the notebooks.
+You can also run all example notebooks as follows:
+
+```console
+git clone https://github.com/cleanlab/examples.git
+```
+
+Then specify your local version of cleanlab source in the first line of: **examples/requirements.txt**
```

Comment:
Can you clarify what this means and give an example? E.g. "specify your local version of cleanlab as a relative path in ..., e.g. `../cleanlab` if `cleanlab` and `examples` are sibling directories.

Reply:
done